scripts/sysman/fabric.yml

```diff
@@ -304,6 +304,26 @@ returns:
     - $X_RESULT_ERROR_INSUFFICIENT_PERMISSIONS:
         - "User does not have permissions to query this telemetry."
 --- #--------------------------------------------------------------------------
+type: function
+desc: "Get Fabric port throughput from multiple ports in a single call"
+class: $sFabricPort
+name: GetMultiPortThroughput
+details:
+    - "The application may call this function from simultaneous threads."
+    - "The implementation of this function should be lock-free."
+params:
```

Comment:
Recommend different ordering and naming: `count`, `phPorts`, `ppThroughputs`

Reply:
change count to numPorts